Files/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -554,22 +554,33 @@ private async void FileList_ItemClick(object sender, ItemClickEventArgs e)
             }
         }
 
-        private async void FileList_ChoosingItemContainer(ListViewBase sender, ChoosingItemContainerEventArgs args)
+        private async void FileList_ContainerContentChanging(ListViewBase sender, ContainerContentChangingEventArgs args)
```

Comment:
Would it be possible (/feasible) to move this into ``BaseLayout`` to reduce code duplication between layouts that utilize a ``ListViewBase`` for the file list? Also is should this change be made in ColumnView(s) as well since it also uses ChoosingItemContainer?

Reply:
> Would it be possible (/feasible) to move this into BaseLayout
I don't think so, all layouts have slightly different logic in the event

> Should this change be made in ColumnView(s) 
It really should but I hate putting my hands into the ColumnView code lol (kidding, I'll get to it)

Reply:
Before doing this for ColumnView I'd like some feedback first though. Does it work? Is it really better than the current solution?

Reply:
Yeah it works great, and I haven't reproduced that icons not loading issue either.

Reply:
Done for column view 👍